docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+services:
+  inv_sig_helper:
+    # build: .
```

Comment:
Please comment instead the docker image from quay registry.

We prefer that this docker-compose file is for development not production.